tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_proactive_IPI.py

```diff
@@ -67,25 +123,36 @@ def test_automated_recovery_from_failed_nodes_IPI_proactive(
         machine.delete_machine(machine_name)
         log.info(f"Successfully deleted machine {machine_name}")
 
-        # Check the pods should be in running state
-        all_pod_obj = pod.get_all_pods(wait=True)
+        # DC app pods on the failed node will get automatically created on
+        # other running node. Waiting for all dc app pod to reach running
+        # state
+        pod.wait_for_dc_app_pods_to_reach_running_state(dc_pod_obj)
+        log.info("All the dc pods reached running state")
+
+        # Check all OCS pods status, they should be in running state
+        all_pod_obj = pod.get_all_pods(
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE
+        )
         for pod_obj in all_pod_obj:
-            if '-1-deploy' and 'ocs-deviceset' not in pod_obj.name:
+            if '-1-deploy' not in pod_obj.name:
                 try:
-                    wait_for_resource_state(
+                    helpers.wait_for_resource_state(
                         resource=pod_obj, state=constants.STATUS_RUNNING,
                         timeout=200
                     )
-                # 'rook-ceph-crashcollector' on the failed node stucks at pending
-                # state. BZ 1810014 tracks it.
-                # Ignoring 'rook-ceph-crashcollector' pod health check as WA and
-                # deleting its deployment so that the pod disappears
-                # Will revert this WA once the BZ is fixed
                 except ResourceWrongStatusException:
+                    # 'rook-ceph-crashcollector' on the failed node stucks at
```

Comment:
@PrasadDesala do we have an issue for that?

Reply:
Yes.  
BZ - https://bugzilla.redhat.com/show_bug.cgi?id=1810014
ocs-ci issue - #1631